common/armies/hfe_armies.txt

```diff
@@ -38,3 +38,310 @@
 # 		}
 # 	}
 # }
+
+hfe_army_high_mechanicus_bodyguards = {
+	defensive = yes
+	is_pop_spawned = no
+	pop_limited = no
+	health = 1.1
+	damage = 2
+	morale = 5
+	has_morale = yes
+	morale_damage = 1.5
+	collateral_damage = 0.9
+	war_exhaustion = 2
+	icon_frame = 5
+	has_species = yes
+
+	resources = {
+		category = armies
+		cost = {
+			trigger = {
+				owner = {
+					NOT = {
+						is_primitive = yes
+						is_country_type = primitive
+					}
+				}
+			}
+			energy = 500
+			food = 30
+			alloys = 300
+		}
+		cost = {
+			alloys = 100
+		}
+		produces = {
+			unity = 1
+		}
+		upkeep = {
+			food = 2
+			alloys = 2
+			energy = 2
+			minerals = 2
+		}
+	}
+
+	potential = {
+		owner = { has_country_flag = hfe_varelse_primitive_0 }
+		owner = {
```

Comment:
Huh. I thought I already fixed that. I will do so now.

Reply:
That was one of the ones I didn't change, because there was no need. I intended the two to be separate in order to emphasize the country flag over the other matters.

Reply:
With one owner switch effect will be same

Reply:
Then why should we bother?

Reply:
Okay, done